### PR TITLE
Integrating get join us api

### PR DESCRIPTION
**Pull request by Aman16-ai**
## Issue that this pull request solves

 Closes: #741 

### Brief description of what is fixed or changed

- fetched all joinus data from the backend.
- showing the data in the cards

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
Screenshot (733)
Screenshot (734)
Screenshot (728)
Screenshot (729)
Screenshot (730)
Screenshot (731)
Screenshot (732)

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: frontend/src/pages/Admin/Components/JoinUs/Card/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+                <h1>{props.content.name}</h1>
+                <h2 className={style['card-detail']}>{props.content.college}</h2>
+                <h3 className={style['card-detail']}>Department : {props.content.department}</h3>
+                <h3 className={style['card-detail']}>Domain : {props.content.interestedDomain}</h3>
+                <h3 className={style["card-detail"]}>Year : {props.content.year}</h3>
+                <h3 className={style['card-detail']}>{props.content.contact}</h3>
+                <p>{props.content.description}</p>
+                <p>hazem@test.com</p>
+                <a className={style['card-link']} href={props.content.linkedin}>Linkdin</a>
+                <IconButton className={style['icon-button']}>
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/JoinUs/Card/Card.jsx:19]: Please keep two bottons/icons(whichever looks good, IMO buttons will look good ) . One for `delete` and another for `edit`

File: frontend/src/pages/Admin/Components/JoinUs/Card/card.module.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    height: auto;
+    width: 15em;
+    margin: 1em;
+    display: inline-block;
+    background-position: left center;
+    transition: all 0.5s ease-in;
+    background-color: #016795;
+    box-shadow: 0.5em 0.5em 0.5em rgb(54, 53, 53);
+  }
+
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/JoinUs/Card/card.module.scss:14]: Please format this file. Each class should be separated by a single blank line and the code should be formatted

File: frontend/src/pages/Admin/Components/JoinUs/joinus.module.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.card-container {
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/JoinUs/joinus.module.scss:1]: Please format this file. Each class should be separated by a single blank line and the code should be formatted

File: frontend/src/pages/Admin/Components/JoinUs/Card/Card.jsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        <div className={style["card-item"]}>
+            <div className={style["card-info"]}>
+                <h1>{props.content.name}</h1>
+                <h2 className={style['card-detail']}>{props.content.college}</h2>
+                <h3 className={style['card-detail']}>Department : {props.content.department}</h3>
+                <h3 className={style['card-detail']}>Domain : {props.content.interestedDomain}</h3>
+                <h3 className={style["card-detail"]}>Year : {props.content.year}</h3>
+                <h3 className={style['card-detail']}>{props.content.contact}</h3>
+                <p>{props.content.description}</p>
+                <p>hazem@test.com</p>
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/JoinUs/Card/Card.jsx:17]: why do we need this hardcoded value? [please remove it, if not required]

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Aman16-ai please incorporate the requested changes

**Review by ank1traj — CHANGES_REQUESTED** [review]

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Aman16-ai The UI https://github.com/HITK-TECH-Community/Community-Website/pull/763 looks better. Can we please modify this one's as well to look the same in ordre to maintain the consistency?

**Comment by Aman16-ai** [comment]
> @Aman16-ai The UI #763 looks better. Can we please modify this one's as well to look the same in ordre to maintain the consistency?

Ok

**Comment by Aman16-ai** [comment]
> > @Aman16-ai The UI #763 looks better. Can we please modify this one's as well to look the same in ordre to maintain the consistency?
> 
> Yes

**Comment by Kajol-Kumari** [comment]
> > > @Aman16-ai The UI #763 looks better. Can we please modify this one's as well to look the same in ordre to maintain the consistency?
> > 
> > 
> > Yes

Please try to reuse the code as much as possible inorder to reduce the redundency

**Comment by Aman16-ai** [comment]
Improved Button u
Screenshot (755)
i

File: frontend/src/pages/Admin/Components/JoinUs/Card/Card.jsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+            <div className={style["card-info"]}>
+                <h1>{props.content.name}</h1>
+                <h2 className={style['card-detail']}>{props.content.college}</h2>
+                <h3 className={style['card-detail']}>Department : {props.content.department}</h3>
+                <h3 className={style['card-detail']}>Domain : {props.content.interestedDomain}</h3>
+                <h3 className={style["card-detail"]}>Year : {props.content.year}</h3>
+                <h3 className={style['card-detail']}>{props.content.contact}</h3>
+                <p>{props.content.description}</p>
+                <p>{props.content.email}</p>
+                <a className={style['card-link']} href={props.content.linkedin}>Linkdin</a>
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/JoinUs/Card/Card.jsx:16]: can we please make the `cursor:pointer` for this as this is an url? Also, on-hover, we should give a different color to increase the visibility

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Aman16-ai one last thing to add on. Rest is nice and clean!

File: frontend/src/pages/Admin/Components/JoinUs/Card/Card.jsx (hunk 5 above)
Comment (Aman16-ai) [reply frontend/src/pages/Admin/Components/JoinUs/Card/Card.jsx:16]: Yes

**Comment by Aman16-ai** [comment]
Cursor pointer and added hover effect
Screenshot (757)
.

**Review by Kajol-Kumari — APPROVED** [review]
Thanks for making the suggested changes :)

LGTM 👍

**Pull request merged**